src/screens/HomeScreen.tsx

```diff
@@ -32,21 +34,34 @@ export const HomeScreen = (navProps: HomeStackScreenProps<'Home'>) => {
 
   const headerLeft = useCallback(
     () => (
-      <HeaderLeftRefreshButton refreshing={refreshing} onRefresh={refresh} />
+      <>
+        <HeaderLeftRefreshButton refreshing={refreshing} onRefresh={refresh} />
+        <AppConfigSwitcherButton />
+      </>
     ),
     [refreshing, refresh],
   );
 
+  const headerLeftAppPicker = useCallback(
+    () => <AppConfigSwitcherButton />,
+    [],
+  );
```

Comment:
I think this would be a breaking change since it will now always show the app switcher. I wonder if we could make it opt in, similar to the refresh button.

Reply:
Indeed, I mean't to remove this from here.

Reply:
Done